kratos/input_output/logger_message.cpp

```diff
@@ -12,6 +12,7 @@
 //
 
 // System includes
+#include <ctime>
```

Comment:
I am quite convinced that there is a more "C++" way of doing this with std::chrono (haven't checked though :) )

Reply:
https://en.cppreference.com/w/cpp/chrono